keda/Chart.yaml

```diff
@@ -8,7 +8,7 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.9.4
```

Comment:
please, don't bump this version. we do it during release process

Reply:
Sure